src/views/create-update-recipe.twig

```diff
@@ -65,6 +72,25 @@
             }
 
 
+            {% if registration_required %}
```

Comment:
Note comment above about how we shouldn't have to handle registration at all in this template.